### PR TITLE
Add portuguese translation

### PR DESCRIPTION
**Pull request by cristianofromagio**
Had some fun lately with Google Apps Script thanks to this amazing project for introducing it to me.

Hope for this project to reach more people, so here we go!

### CODE REVIEW

**Comment by iteles** [comment]
@cristianofromagio Thank you for contributing this! 👀 

@mckennapsean @nelsonic I'm happy to review this one today, will get to it before the end of the day 👍

**Review by nelsonic — COMMENTED** [review]
@iteles yeah, happy for you to review+merge given your fluency in PT. 👍

File: README.pt.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+No entanto, você pode modificar isso, por meio do **Editor de script**. A linha:
+
+```javascript
+result += "<h4 style='text-transform: capitalize; margin-bottom: 0'>" + key + "</h4><div>" + obj[key] + "</div>";
+```
+
+tem tudo que você precisa. Você pode ajustar a marcação para se adequar à sua necessidade. Escolhemos um `<h4>` porque era o melhor tamanho para o e-mail e adicionamos uma pequena quantidade de CSS para corrigir a capitalização (as letras são todas minúsculas no objeto JS) e um pouco de espaçamento por padrão. Embora estilos inline como estes sejam geralmente má prática em páginas web normais, para um HTML no e-mail eles são a única maneira confiável de usar CSS!
+Optamos por um `<div>` para a parte do valor, porque ele poderia ser qualquer coisa - uma linha só, múltiplas linhas (um `<p>` por exemplo não seria suficiente).
+
+While we're here, there's also a `replyTo` option for the `sendEmail()` method which is commented out by default:
```
Comment (iteles) [inline README.pt.md:215]:
```suggestion
```

Comment (iteles) [reply README.pt.md:215]: English portion left in by mistake, translation already below.

File: README.pt.md [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+### 14. Adicione a função `record_data` no seu Google Apps Script
+
+![record_data example](https://cloud.githubusercontent.com/assets/194400/10581613/8b4f9ad4-767b-11e5-90cc-962a9d6acc91.png)
+
+Isso irá salvar os dados recebidos do `POST` como uma *linha* na planilha.
+Consulte: [**google-apps-script.js**](google-apps-script.js) para o código completo que você pode *copiar e colar*.
+
+### 15. Salve uma nova versão e publique-a novamente
+
+Siga os passos 4, 5 e 6 para salvar uma nova versão e ***publicar novamente*** (***re-publish***) o script.
```
Comment (iteles) [inline README.pt.md:248]: 👍

File: README.pt.md [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+
+Para sua conveniência, hospedamos uma demonstração do formulário no GitHub Pages,
+confira para ver o código e como funciona:
+https://dwyl.github.io/learn-to-send-email-via-google-script-html-no-server/
+
+
+## Adicione seus próprios campos!
+
+Em resposta à [solicitação de Henry Beary](https://github.com/dwyl/learn-to-send-email-via-google-script-html-no-server/issues/9),
+tornamos o tratamento do formulário *genérico*, o que significa que agora você pode adicionar quaisquer campos desejar ao formulário.
```
Comment (iteles) [inline README.pt.md:305]:
```suggestion
tornamos o tratamento do formulário *genérico*, o que significa que agora você pode adicionar quaisquer campos que desejar ao formulário.
```

**Review by iteles — APPROVED** [review]
Great translation, thank you @cristianofromagio !!

@nelsonic I've added a couple of very minor suggested changes - could you please review/merge? Thanks!

**Review by nelsonic — APPROVED** [review]
@cristianofromagio thanks again for this superb PR. 🎉 
I'm sure a lot of people will find it useful. 🇧🇷 + 🇵🇹 + 🇲🇿, etc. 👍 
@iteles thanks for the extended review and updates. 👩‍💻 ❤️

File: README.ko.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Google Apps Mail을 사용해 **정적** HTML Form에서 메일을 보내세요!
 
-**Language : [English](https://github.com/dwyl/learn-to-send-email-via-google-script-html-no-server#send-email-from-a-static-html-form-using-google-apps-mail) | `한국어` | [Español](https://github.com/dwyl/learn-to-send-email-via-google-script-html-no-server/blob/master/README.es.md#older-translation)**
+**Language : [English](https://github.com/dwyl/learn-to-send-email-via-google-script-html-no-server#send-email-from-a-static-html-form-using-google-apps-mail) | `한국어` | [Español](https://github.com/dwyl/learn-to-send-email-via-google-script-html-no-server/blob/master/README.es.md#older-translation) | [Português](https://github.com/dwyl/learn-to-send-email-via-google-script-html-no-server/blob/master/README.pt.md)**
```
[the remark this answers is not in the document]
Comment (nelsonic) [reply README.ko.md:3]: Agreed. I think we should remove them in a _separate_ (_future_) PR. 👍

File: README.pt.md [hunk 5]
```diff
@@ -0,0 +1,370 @@
...
+
+## *Como*?
+
+### 1. Faça uma cópia da planilha de modelo
+
+> Sample: https://docs.google.com/spreadsheets/d/1Bn4m6iA_Xch1zzhNvo_6CoQWqOAgwwkOWJKC-phHx2Q/copy
+
+Faça login na sua conta Google e clique em "**Fazer uma cópia**..."
+
+![1-make-copy](https://user-images.githubusercontent.com/1406149/29245471-feb7b034-7f97-11e7-9c0d-f06238e8362b.png)
```
[the remark this answers is not in the document]
Comment (cristianofromagio) [reply README.pt.md:50]: Yep, I followed the steps and created a copy of the documents myself. I translated all *written* dialogue labels as their matching on my interface (brazillian portuguese) although I don't think there will be much difference between variants.

About updating the screenshots, at the moment I think the highlights on them plus the text are enough. The hightlights are really helpful to guide the user on the interface (no need to "search" the element throughout the page).

File: README.pt.md [hunk 6]
```diff
@@ -0,0 +1,370 @@
...
+
+Se você não quer aceitar esse risco em potencial, descomente o código da variável `TO_ADDRESS` e defina este valor igual ao e-mail que deverá receber os dados do formulário ao ser enviado.
+
+![3-script-editor-showing-script](https://cloud.githubusercontent.com/assets/194400/10560379/9efa5b3a-7501-11e5-96ba-a9e3b2d77ee4.png)
+
+### 4. Salve uma *nova versão* do seu Script
+
+Não é imediatamente *óbvio* mas você deve *clicar* em "**Arquivo**" > "**Gerenciar versões...**"
+**_Nota de tradução:_** pode ser que a interface em `https://script.google.com/` não esteja traduzida, mesmo se seu idioma esteja definido como Português por padrão, o caminho para salvar a nova versão então seria (em inglês): "**File**" > "**Manage versions...**"
```
[the remark this answers is not in the document]
Comment (cristianofromagio) [reply README.pt.md:79]: Like I said above, I tried to follow the steps and match the labels on my (portuguese) interface. But just couldn't get `https://script.google.com` to display on portuguese (or any other than the english default), so I translated as it appeared on similar pages and kept the original on the translator’s note (more like a "translation's note" but you know what I mean 🤣).

**Comment by cristianofromagio** [comment]
Thanks @nelsonic with the technicalities and really thank you @iteles for the thoughtful review!
And thanks you all for creating @dwyl and making it this one-of-a-kind learning place.

Would feel really honored if you could label this PR `hacktoberfest-accepted` if you feel it is worth it! 🙏🏻

**Pull request merged**